app/scripts/controllers/newfromtemplate.js

```diff
@@ -355,6 +354,13 @@ angular.module('openshiftConsole')
             $scope.parameterDisplayNames[parameter.name] = parameter.displayName || parameter.name;
           });
 
+          _.each($scope.template.parameters, function(parameter) {
+            var found = _.find(JSON.parse($routeParams.templateParamsMap), function(param){
+              return param.name === parameter.name;
+            });
```

Comment:
> Do we want to restrict to only name and value values?

Yeah, I don't think they should be able to change anything other than the value for an existing parameter. We should probably show an error if they give the name that does not match a param in the template.